src/app/services/wallet.service.ts

```diff
@@ -35,6 +37,10 @@ export class WalletService {
     return this.wallets.asObservable().map(wallets => wallets ? wallets : []);
   }
 
+  getBalancesUpdated(): Observable<Date> {
```

Comment:
This does not follow the style that has been used in `WalletService`. It would be better if it changed to `get balancesUpdateDate()`